src/usecases/refreshToken.test.js

```diff
@@ -0,0 +1,52 @@
+import refreshToken from "./refreshToken";
+import moment from "moment";
+
+describe("verifyAdminToken", () => {
+  it("it doesn't refresh the token if outside the refresh timeframe", () => {
+    const authenticationToken = {
+      type: "admin",
+      exp: moment().subtract(4, "hours").unix(),
+    };
+
+    const tokenProvider = {
+      validate: jest.fn(() => authenticationToken),
+      generate: jest.fn(() => "encodedToken"),
+    };
+    const container = {
+      getTokenProvider: () => tokenProvider,
+    };
+    const {
+      refreshedToken,
+      refreshedEncodedToken,
+      isTokenRefreshed,
+    } = refreshToken(container)(authenticationToken);
+
+    expect(isTokenRefreshed).toEqual(false);
+    expect(refreshedToken).toBeUndefined;
+    expect(refreshedEncodedToken).toBeUndefined;
```

Comment:
With our other usecases we still return the key but `null`. What do you think about following suit here?